dashboard/management/commands/term.py

```diff
@@ -2,7 +2,7 @@
 from dashboard.models import AcademicTerms
 from dashboard.common.db_util import canvas_id_to_incremented_id
 from datetime import datetime
-import pytz
+from zoneinfo import ZoneInfo
 
```

Comment:
Some how This package https://github.com/adamchainz/django-upgrade did not removed this, so i had to do this manually.

Reply:
May want to add the dependency upgrade instruction into https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/docs/CONTRIBUTING.md, and include this package there

Reply:
Done, added the dependency section upgrade info https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1554/commits/ca01926cd8de9537ed3c7edf31d2288e20e641f6